defos/src/defos_html5.cpp

```diff
@@ -1,11 +1,8 @@
-#ifndef DLIB_LOG_DOMAIN
-#define DLIB_LOG_DOMAIN "defos"
-#endif
-#include <dmsdk/sdk.h>
 
-#if defined(DM_PLATFORM_HTML5)
+#if defined(DM_PLATFORM_HTML5) && !defined(DM_HEADLESS)
```

Comment:
The `DLIB_LOG_DOMAIN` is automatically set by the extension builder.

Reply:
It was there for compatibility with *really old* versions of Defold where that didn't happen, but I guess it's ok to remove it now.